.github/workflows/release-embedding-sdk.yml

```diff
@@ -191,3 +226,29 @@ jobs:
           cd sdk
           echo //registry.npmjs.org/:_authToken=${{ secrets.NPM_RELEASE_TOKEN }} > .npmrc
           npm publish
+
+  git-tag:
```

Comment:
Moved the git tag step last following the [recommended workflow.](https://github.com/metabase/metabase/issues/42749#issuecomment-2137767536)